scoop-basics/src/main/java/com/example/scoop/basics/ui/layoutsample/screen/LayoutScreen.java

```diff
@@ -0,0 +1,9 @@
+package com.example.scoop.basics.ui.layoutsample.screen;
+
+import com.example.scoop.basics.R;
+import com.lyft.scoop.Layout;
+import com.lyft.scoop.Screen;
+
+@Layout(R.layout.layout_sample)
```

Comment:
I was actually testing if you can create a screen without a module which is technically a new feature. That is why I excluded a module.